src/10-ipfs-0-4-0-released/index.md

```diff
@@ -0,0 +1,121 @@
+---
+date: 2016-03-17
+id: 10-ipfs-0-4-0-released
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+- {name: "10-ipfs-0-4-0-released", link: "./" }
+tags: modules
+title: IPFS 0.4.0 has been released
+author: Kyle Drake and Whyrusleeping
```

Comment:
I dislike using my real name on the internet, on github, irc, any forums or whatever i prefer to be called whyrusleeping